Source/GTMSessionFetcher.m

```diff
@@ -134,6 +134,11 @@
 #define GTM_SDK_REQUIRES_SECTRUSTEVALUATEWITHERROR 0
 #endif
 
+#if __has_attribute(swift_async)
+// Once Clang 13/Xcode 13 can be assumed, can switch to NS_SWIFT_DISABLE_ASYNC.
+#define GTM_SWIFT_DISABLE_ASYNC __attribute__((swift_async(none)))
+#endif
```

Comment:
You need the else to define it away, no?

Reply:
Oops, yep. That's what happens when I switch from defining the NS_SWIFT_… symbol as empty if it's not defined to defining my own thing… gotta remember the else!